mithril-client/src/commands/snapshot/show.rs

```diff
@@ -55,6 +55,14 @@ fn convert_to_field_items(snapshot_message: &SnapshotMessage) -> Vec<SnapshotFie
         ),
         SnapshotFieldItem::new("Digest".to_string(), snapshot_message.digest.to_string()),
         SnapshotFieldItem::new("Size".to_string(), format!("{}", snapshot_message.size)),
+        SnapshotFieldItem::new(
+            "Node version".to_string(),
```

Comment:
Small modification to avoid confusing Mithril node and Cardano node:
```suggestion
            "Cardano node version".to_string(),
```